README.md

```diff
@@ -43,21 +43,21 @@ npm install
 
 ## New Token Methods
 
-Líf token is ERC20 compatible but it also has two more methods to allow the transference of data between users/contracts.
+Líf token is ERC20 compatible but it also has two more methods to allow the transference of data and execution of calls between users/contracts.
 
 ### transferData
 
 Transfer tokens from one address to another.
 ```
-transferData(address _to, uint _value, string _data) => (bool success)
+transferData(address to, uint value, bytes data, bool doCall) => (bool success)
```

Comment:
Because I remember a week ago when I was working on the contract I could deploy the contract because gasLimit and I had to stop adding new things and do a refactor of the code to save space, maybe now we cant, but it I think is calculated by: CONTRACT_DATA_BYTES * GAS_PRICE, this has to be less than GAS_LIMIT, with is now near 4 million wei. You can now how much it cost seeing the gasUsed on a contract deployment tx.
In a near future zeppelin libs are planned to be hosted on the blockchain, and that will help up save space.
Although I been thinking on develop another smart contract to just do the crowdsale stage, this can also be a solution if we need to save gas in a future.